modules/intentIqAnalyticsAdapter.js

```diff
@@ -295,6 +278,7 @@ function defineGlobalVariableName() {
   const iiqConfig = getIntentIqConfig();
```

Comment:
This will be undefined when the IIQ Id Module is not present in our `off` group. It will not allow for manual reporting or processing since the global namespace is not set properly.

I suggest you add the handling to this function to allow users to pass the id params to the analytics module options.